# SPDX-FileCopyrightText: 2023 German Aerospace Center (DLR)
# SPDX-License-Identifier: Apache-2.0

#in ms
float32 time_of_week  
float32 lon	
float32 lat
#in m
float32 height_above_ellipsoid
#in m
float32 height_above_mean_sea_level
#in m
float32 horizontal_accuracy_estimate
#in m
float32 vertical_accuracy_estimate
# test arra
float32[3] data_3
float32[2] data_n
